resources/sip11/ra/src/test/java/org/mobicents/slee/resource/sip11/test/SipResourceAdaptorTest.java

```diff
@@ -0,0 +1,144 @@
+package org.mobicents.slee.resource.sip11.test;
+
+import javax.sip.ListeningPoint;
+import javax.slee.facilities.Tracer;
+import javax.slee.resource.ConfigProperties;
+import javax.slee.resource.ConfigProperties.Property;
+import javax.slee.resource.ResourceAdaptorContext;
+
+import org.mobicents.ha.javax.sip.ClusteredSipStack;
+import org.mobicents.slee.resource.sip11.SipResourceAdaptor;
+import org.mobicents.slee.resource.sip11.SleeSipProviderImpl;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.BeforeClass;
+
+import org.powermock.reflect.Whitebox;
+
+import static org.mockito.Matchers.*;
+import static org.mockito.Mockito.*;
+import org.mockito.stubbing.*;
+import org.mockito.invocation.*;
+import java.util.Arrays;
+import java.util.Iterator;
+
+public class SipResourceAdaptorTest {
+
+    private  static final int SIP_RA1_PORT = 5060;
+	private static final int SIP_RA2_PORT = 5059;
+	private static final String STACK_ADDRESS = "127.0.0.1";
+    // common mocked tracer
+	private static Tracer tracer1 = mock(Tracer.class);
```

Comment:
Tracer is used internally in SipResourceAdaptor.java in raConfigure(). If setResourceAdaptorContext used, then still some Tracer implementation is needed.